ivre/utils.py

```diff
@@ -1785,3 +1822,35 @@ def key_sort_none(value):
 
         """
         return value
+
+
+def ptr2addr(ptr):
+    """
+    Returns the IP address (v4 or v6) represented by the given PTR,
+    None if the string does not seem to be a PTR
+    """
+    if ptr.endswith(".in-addr.arpa"):
+        return '.'.join(reversed(ptr.split(".")[:4]))
+    elif ptr.endswith(".ip6.arpa"):
+        return int2ip6(int(ptr[:-9].replace('.', '')[::-1], 16))
+    return None
+
+
+def is_ptr(ptr):
+    """
+    Check whether the given string is a PTR
+    """
+    return ptr.endswith(".in-addr.arpa") or ptr.endswith(".ip6.arpa")
+
+
+def deep_sort_dict_list(elt):
+    """
+    Deep sort the list values inside a dictionary.
+    elt must be a dictionary
+    Notice: It does not sort nested lists.
+    """
+    for key, value in viewitems(elt):
```

Comment:
Since you do not use `key` (as spotted by codacy), you can replace that with `viewvalues` (that you need to import):
```suggestion
    for value in viewvalues(elt):
```